cachito/web/config.py

```diff
@@ -11,6 +11,7 @@
 class Config(object):
     """The base Cachito Flask configuration."""
 
+    DEBUG = False
```

Comment:
We may need to set `DEBUG` to True in `DevelopmentConfig` accordingly.

Reply:
And we can remove this config from the `ProductionConfig class`, since it became redundant. (L44)